### ISSUE TITLE
Lernstatistik: Optionen sind merkwürdig sortiert 

### ISSUE THREAD

**Issue by alinastolzenburg** [elan-ev/studip-lernkarten#33]
Außerdem ist der Zusammenhang zu den Optionen beim Lernen der Karten etwas unklar. Wird gewusst und teilweise gewusst zu erneut zu lerndend?

![image](https://github.com/elan-ev/studip-lernkarten/assets/116811905/a305f6fe-31c6-42dc-beed-49940cec190d)

**Comment by luniki** [comment]
Ja, "Erneut zu lernende Karte" ist eine Karte, die vorher eine "Gelernte Karte" war, dann aber vergessen wurde. Alternativ kann ich deren Anteil aber auch den "Zu lernenden Karten" zuschlagen.  Bringt das etwas?
